gh-pages/content/user-guides/lib-author/configuration/targets/go.md

```diff
@@ -3,15 +3,31 @@
 !!! danger
     The **go** target is currently unstable and not suitable for production use.
 
-To enable go package generation, add the **go** key with an empty object to the jsii targets configuration.
+To enable go package generation, add the `go` key to the jsii targets configuration:
 
-This will add generated go package code to your specified `outDir` for testing and experimentation.
+- `packageName` (optional) - The name of the Go package name. If this is not
+  specified, the package name will be derived from the npm module name by
+  filtering non-alphanumeric characters (e.g. `aws-cdk-lib` will be
+  `awscdklib`). You can use this field to set a specific package name.
+- `moduleName` (required) - The name of the **target repository** in which this
```

Comment:
Maybe just rename to `repositoryName`?

Reply:
We wanted to keep this change to a minimum since this is consumed by downstream tools like jsii-release, so decided against changing the current semantics. We can evolve this at a later stage. I understand it's not ideal.

Reply:
Fair enough